src/app/scheduling/datepicker/reactive-form/reactive-form.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+    selector: 'app-datetimepicker-reactive-form',
+    styleUrls: ['./reactive-form.component.scss'],
+    templateUrl: './reactive-form.component.html'
+})
+export class DateTimePickerRFSampleComponent implements OnInit {
+    public dateTimeForm: FormGroup;
+    public date = new Date();
+    public minDate = new Date(new Date(this.date.getFullYear(), this.date.getMonth(), this.date.getDate() - 10));
+    public maxDate = new Date(new Date(this.date.getFullYear(), this.date.getMonth(), this.date.getDate() + 15));
+    public minTime = '06:15:30';
+    public maxTime = '15:15:30';
+
+    constructor(fb: FormBuilder) {
+        this.dateTimeForm = fb.group({
+            date: [this.date, Validators.required],
+            time: [this.date, Validators.required]
+        });
+    }
+
+    ngOnInit() {
```

Comment:
You don't actually need any of this. Since both pickers emit `valueChange` you can do the pretty much the same thing in template, something like this:
```html
<igx-date-picker 
    #datePicker 
    (valueChange)="timePicker.value = datePicker.value" 
    [value]="date">
</igx-date-picker>

<igx-time-picker 
    #timePicker
    (valueChange)="datePicker.value = timePicker.value"
    [value]="date'>
</igx-time-picker>
```
That's just another way of doing it without having to implement `OnInit` and accessing the form group in the hook. You can do either, of these but if you're sticking with the `OnInit` approach please make sure to add a little access modifier, just to be lint-proof. :)